guide/principles/principles.md

```diff
@@ -0,0 +1,182 @@
+
+These are principles we in the Bitcoin Design Community have identified and stand behind. Some of these come from the technology itself, and others from the behavior and ethos of the community. Although every use case and product is different, applications should strive to follow these principles. Diverging from them should only be done with very good reason.
+
+
+- [Self-custody](#self-custody)
+- [Security](#security)
+- [Inclusion](#inclusion)
+- [Interoperability](#interoperability)
+- [Transparency](#transparency)
+- [Privacy](#privacy)
+- [Decentralization](#decentralization)
+
+---
+
+## Self-custody
+
+> Let users control their private keys, with no risk for seizure or freezing of funds
+
+Our existing mental models of access to digital services are usernames and passwords, controlled by a company that has custody of your funds and data. With everyone having direct access to the Bitcoin network without middlemen, there is no longer a need for people to delegate control of their funds. While it comes with greater responsibility, it is self-custody that enables the open financial system of peer-to-peer transactions.
+
+**Do**
+- Let users control their private keys and funds directly
+- Create an easy path to self-custody for Bitcoin beginners
+
+**Don't**
+- Custody funds for your users
+- Build products where the users' funds can be seized, or frozen
+
+---
+
+## Security
+
+> Provide appropriate and progressive security for all types of users
+
+Self-custody often leaves the end-user responsible for the security of their private keys. They can only do a good job of that if we provide them with appropriate tools and awareness of best practices. 
+
+Security is especially important when onboarding people new to Bitcoin where solutions that work for advanced users might backfire. New users are likely to start by storing small amounts until they get comfortable with self-custody. The concept of progressive security is a good idea for here. This would let a user upgrade their security and private key management scheme as their savings grow. 
+
+Education and awareness is a big part of security, protecting users from bad actors and potentially their own bad behavior. It is unrealistic to expect beginners to take in all the knowledge acquired by advanced users in one go, for example while [onboarding]({{ '/guide/onboarding/introduction/' | relative_url }}) to a bitcoin product. We should therefore consider how to continuously educate and level up user awareness of best practices and risks.
+
+Security can also be a feeling. A product that is polished, good looking, easy to use and transparently communicates how it works can help users feel more secure. Especialy compared to one with the exact same security measures that lack these qualities.
+
+
+**Do**
+- Take safeguarding of users funds seriously
```

Comment:
```suggestion
- Take safeguarding of user funds seriously
```

Reply:
I think it should either be,
- ... users' funds seriously
- ... user funds seriously

Reply:
Resolved to ... users' funds seriously, for now